### ISSUE TITLE
Add autoreboot option to postinst script

### ISSUE THREAD

**Issue by louib** [mvo5/unattended-upgrades#252]
I feel like the `Unattended-Upgrade::Automatic-Reboot` option is important enough to also be present in the postinst script.

**Comment by rbalint** [comment]
If you mean asking the user in a debconf question if this should be enabled then I don't think adding this to postinst would be a good idea. The script is complicated enough already and this option is for advanced users who can find the option themselves or install a config snippet easily.

**Comment by louib** [comment]
My bad, I think it's actually the `config` script I'm referring to. [Currently there's only one question asked when configuring u-u using `dpkg-reconfigure`](https://github.com/mvo5/unattended-upgrades/blob/e25d22c1791e2fe6d2086926f4847b90197cc583/debian/config#L19). My suggestion would be to add the auto reboot option in the configuration script, while preserving the current default. This way the default stays the same, but this IMO important option would gain visibility.

**Comment by rbalint** [comment]
Yes, I understood the idea that way and I don't think adding the question would be a good idea.
